tests/show_techsupport/test_techsupport.py

```diff
@@ -457,3 +483,64 @@ def test_techsupport_commands(
         )
 
     pytest_assert(len(cmd_not_found) == 0, cmd_not_found)
+
+def test_secret_removed_from_show_techsupport(
+    duthosts, enum_rand_one_per_hwsku_hostname, creds_all_duts, check_image_version, setup_password
+):
+    """
+    This test checks following secrets been removed from show techsupport result:
+        Tacacs key
+        Radius key
+        snmp community string
+        /etc/shadow, which includes the hash of local/domain users' password
+        /etc/sonic/*.certs, which are Azure internal used certificates
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    tacacs_passkey = creds_all_duts[duthost]['tacacs_passkey']
+    radius_passkey = creds_all_duts[duthost]['radius_passkey']
+    snmp_rocommunity = creds_all_duts[duthost]['snmp_rocommunity']
```

Comment:
>snmp_rocommunity [](http://example.com/codeflow?start=4&length=16)

also check snmp_rwcommunity ?

Reply:
There is no snmp_rwcommunity in the testbed config files: 
https://github.com/Azure/sonic-mgmt/blob/cf70a2400dfa8308c123a21ec4eda494d27fc1d8/ansible/group_vars/lab/lab.yml